test/lib/command-helpers/element-utils/common.js

```diff
@@ -242,24 +243,51 @@ describe('common "element-utils" helper', () => {
             it('should correctly calc with multiply top toolbar height by passed pixel ratio', async () => {
                 utils.getTopToolbarHeight.withArgs(browser).returns(2);
 
-                const {top} = await utils.calcWebViewCoords(browser, {pixelRatio: 2});
+                const {top} = await utils.calcWebViewCoordsNative(browser, {pixelRatio: 2});
 
                 assert.equal(top, 4);
             });
 
             it('should set to zero if calculated coord is negative', async () => {
                 utils.getTopToolbarHeight.withArgs(browser).returns(-10);
 
-                const {top} = await utils.calcWebViewCoords(browser, {pixelRatio: 1});
+                const {top} = await utils.calcWebViewCoordsNative(browser, {pixelRatio: 1});
 
                 assert.equal(top, 0);
             });
         });
 
         it('should return "width", "height", "left" and "top" coords', async () => {
-            const coords = await utils.calcWebViewCoords(browser);
+            const coords = await utils.calcWebViewCoordsNative(browser);
 
             assert.hasAllKeys(coords, ['width', 'height', 'left', 'top']);
         });
     });
+
+    describe('"calcWebViewCoordsNative" method', () => {
+        it('should use _nativeElementsSize if exists', async () => {
+            sinon.stub(utils, '_nativeElementsSize').value({
```

Comment:
We don't use private fields in test files. Here you just need to call `mkUtilsStub` with the necessary arguments (send `nativeElementsSize` as second argument`).

I didn't notice this problem at the first review.